internal/elasticsearch/index/index_test.go

```diff
@@ -0,0 +1,127 @@
+package index_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/elastic/terraform-provider-elasticstack/internal/acctest"
+	"github.com/elastic/terraform-provider-elasticstack/internal/clients"
+	sdkacctest "github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
```

Comment:
Lol at first I thought this was a typo then I realized it's to override the import name. Can you tell I'm not a Golang developer 😁